media/js/firefox/australis/fx38_0_5/firstrun.js

```diff
@@ -83,14 +11,14 @@
             $document = $(document);
 
             // signup CTA opens about:accounts
-            $ctaSignup.on('click', function(e) {
+            $('#cta-signup').on('click', function(e) {
                 e.preventDefault();
 
                 Mozilla.UITour.showFirefoxAccounts();
             });
 
             // signin CTA opens hamburger menu
-            $ctaSignin.on('click', function(e) {
+            $('#cta-signin').on('click', function(e) {
```

Comment:
Since they're only being used once, I figured we didn't need to put them in variables. I'm happy to go back to `var`'s if you think that's better.